KDCircularProgress/KDCircularProgress.swift

```diff
@@ -169,6 +169,12 @@ public class KDCircularProgress: UIView, CAAnimationDelegate {
         }
     }
     
+    @IBInspectable public var progressLayerRadiusPadding: CGFloat = 1 {
+        didSet {
+            progressLayer.progressLayerRadiusPadding = progressLayerRadiusPadding
```

Comment:
Don't we also want to clamp this to some value?

Reply:
Good catch! Updating this as well.